### PR TITLE
refactor(deepl): cleanup

### PR DESCRIPTION
**Pull request by isabelroses**
## 🔧 What does this fix? 🔧

TODO:
- [] remove some unneeded css

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/deepl/catppuccin.user.css [hunk 1]
```diff
@@ -100,6 +100,7 @@
       background-color: @mantle;
     }
 
+    html,
```
Comment (uncenter) [inline styles/deepl/catppuccin.user.css:103]:
```suggestion
    &,
```

File: styles/deepl/catppuccin.user.css [hunk 2]
```diff
@@ -259,10 +260,18 @@
     }
 
     // header
-    [class^="BasePageHeader-module--container"] {
+    .BasePageHeader-module--container--2By68 {
```
Comment (uncenter) [inline styles/deepl/catppuccin.user.css:263]: Why change this? The previous version was more resistant to change as it still works even if the autogenerated bit at the end is different.

File: styles/deepl/catppuccin.user.css [hunk 3]
```diff
@@ -753,5 +758,4 @@
   @macchiato: { @rosewater: #f4dbd6; @flamingo: #f0c6c6; @pink: #f5bde6; @mauve: #c6a0f6; @red: #ed8796; @maroon: #ee99a0; @peach: #f5a97f; @yellow: #eed49f; @green: #a6da95; @teal: #8bd5ca; @sky: #91d7e3; @sapphire: #7dc4e4; @blue: #8aadf4; @lavender: #b7bdf8; @text: #cad3f5; @subtext1: #b8c0e0; @subtext0: #a5adcb; @overlay2: #939ab7; @overlay1: #8087a2; @overlay0: #6e738d; @surface2: #5b6078; @surface1: #494d64; @surface0: #363a4f; @base: #24273a; @mantle: #1e2030; @crust: #181926; };
   @mocha:     { @rosewater: #f5e0dc; @flamingo: #f2cdcd; @pink: #f5c2e7; @mauve: #cba6f7; @red: #f38ba8; @maroon: #eba0ac; @peach: #fab387; @yellow: #f9e2af; @green: #a6e3a1; @teal: #94e2d5; @sky: #89dceb; @sapphire: #74c7ec; @blue: #89b4fa; @lavender: #b4befe; @text: #cdd6f4; @subtext1: #bac2de; @subtext0: #a6adc8; @overlay2: #9399b2; @overlay1: #7f849c; @overlay0: #6c7086; @surface2: #585b70; @surface1: #45475a; @surface0: #313244; @base: #1e1e2e; @mantle: #181825; @crust: #11111b; };
 }
-
 // vim:ft=less
```
Comment (uncenter) [inline styles/deepl/catppuccin.user.css:761]:
```suggestion

// vim:ft=less
```

File: styles/deepl/catppuccin.user.css [hunk 4]
```diff
@@ -709,11 +715,13 @@
       }
     }
 
+    // whatever
     .lmt__glossary_ad_checkmark {
       border-bottom-color: @green;
       border-right-color: @green;
     }
 
+    // zen mode
```
Comment (uncenter) [inline styles/deepl/catppuccin.user.css:724]:
```suggestion
    .lmt__glossary_ad_checkmark {
      border-bottom-color: @green;
      border-right-color: @green;
    }

```

File: styles/deepl/catppuccin.user.css (hunk 2 above)
Comment (isabelroses) [reply styles/deepl/catppuccin.user.css:263]: the previous one didn't seem to work when I was testing.

**Review by watatomo — APPROVED** [review]

File: styles/deepl/catppuccin.user.css (hunk 2 above)
Comment (watatomo) [reply styles/deepl/catppuccin.user.css:263]: could probably just make it `[class*="BasePageHeader-module--"]`. it worked on my end before but ig they changed stuff

**Review by watatomo — APPROVED** [review]

**Pull request merged**